wpiunits/src/test/java/edu/wpi/first/units/EncoderTest.java

```diff
@@ -54,8 +54,10 @@ void testAsDistance() {
     // distance per tick = distance per rotation / ticks per rotation
     var wheelDiameter = Inches.of(6);
     var gearRatio = Value.of(10); // 10:1 ratio
+    @SuppressWarnings("unchecked")
     Measure<Distance> distancePerPulse =
-        wheelDiameter.times(Math.PI).divide(gearRatio).divide(ticksPerRevolution);
+        (Measure<Distance>)
```

Comment:
Github wasn't showing the definition for `tickPerRevolution` earlier. Changing it to a regular `double` would be fine to avoid the cast. I had thought it was a `Per<Dimensionless, Angle>` ratio